CodeEditor/.github/workflows/swift.yml

```diff
@@ -0,0 +1,22 @@
+name: Build and Test
+
+on:
```

Comment:
Just out of interest, what does this file do?

Reply:
This might be a file from the original CodeEditor Repository. I think this is garbage now, I'll take a look